### ISSUE TITLE
Support Vue.js 3

### ISSUE THREAD

**Issue by dmarcoux** [vitogit/vue-chessboard#19]
Hey,

I would like to use vue-chessboard with Vue.js 3.

I'm willing to help out with this if needed. Would you accept a PR for this?

Thank you for creating vue-chessboard!

**Comment by vitogit** [comment]
Yes @dmarcoux if you can create a PR and keep compatibility with older versions, that would be great.
